From fdfb2fb2912f62b153d90ef103322064f86b0268 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 9 Jun 2018 16:16:19 +0300
Subject: [PATCH 4/6] power: Remove unused ondemand related functions

* They were removed in commit fe93c1d21cb9c2802e9abe037b04c2d6cf395fda

Change-Id: I0ae4aa60912be1eb0e1b6235d8e842d5d1f39afa
---
 utils.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/utils.h b/utils.h
index e443749..5af4d16 100644
--- a/utils.h
+++ b/utils.h
@@ -36,10 +36,6 @@ int get_scaling_governor_check_cores(char governor[], int size,int core_num);
 int is_interactive_governor(char*);
 int is_ondemand_governor(char*);
 
-void vote_ondemand_io_busy_off();
-void unvote_ondemand_io_busy_off();
-void vote_ondemand_sdf_low();
-void unvote_ondemand_sdf_low();
 void perform_hint_action(int hint_id, int resource_values[],
     int num_resources);
 void undo_hint_action(int hint_id);
-- 
2.17.1

